client/src/featureform/register.py

```diff
@@ -1590,7 +1591,97 @@ class Customer:
             properties=properties,
         )
 
+### Problems: the user is forced to specify th entity column. If they put the same entity name as the class that they are defining the multiple features in, will it error? 
+### If the mention a different class name, then will it get applied and work?
+class MultiFeatureColumnResource(ColumnResource):
+    def __init__(
+        self,
+        dataset: SourceVariant,
+        df: pd.DataFrame,
+        entity_column: Union[Entity, str] = "",
+        variant: str = "",
+        owner: str = "",
+        inference_store: Union[str, OnlineProvider, FileStoreProvider] = "",
+        timestamp_column: str = "",
+        include_columns: List[str] = [],
+        exclude_columns: List[str] = [],
+        description: str = "",
+        schedule: str = "",
+        tags: List[str] = [],
+        properties: Dict[str, str] = {},
```

Comment:
we're missing the super.__init__() call. A lot of these parameters are unused. Is that intentional? You can configure your IDE to tell you these things. (Not sure how to do in vscode)